conf.yaml

```diff
@@ -782,6 +783,18 @@ contents:
                   -
                     repo:   apm-agent-python
                     path:   docs
+              -
+                title:      APM Ruby Agent
+                prefix:     python
+                current:    1.x
+                branches:   [ master ]
```

Comment:
You've set `current` to 1.x, but you haven't included 1.x in `branches`. It should say:

```
branches: [ master, 1.x ]
```

This assumes that a 1.x branch exists. If you simply want to rename master in the drop-down list that appears in the published docs, there's a way to do that, too. I can't remember exactly how to do that, but I'll dig throguh my notes if that's what you need.

Reply:
Aha, so 1.x does not exist. I'm guessing you want the label to say 1.x. I need to poke around a bit because I think our method for setting the label has changed since I last did that.

Reply:
Looking at previous examples, I think you just need to use  `{ master: 1.x}` instead of `master` when you set `branches`. So you would have: 

```
current:    master
branches:   [ { master: 1.x } ]
```